STHUD/STHUDNewShinyHostView.m

```diff
@@ -134,6 +179,23 @@ - (BOOL)removeHUD:(STHUD *)hud {
 	return YES;
 }
 
+- (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context {
+	if (context == &_hudViewsByNonretainedHUD) {
+		STHUD * const hud = object;
+		id const hudViewKey = [NSValue valueWithNonretainedObject:hud];
+		STHUDNewShinyHUDView * const hudView = [_hudViewsByNonretainedHUD objectForKey:hudViewKey];
+		
+		if ([@"title" isEqualToString:keyPath]) {
+			[UIView animateWithDuration:kSTHUDNewShinyHostViewHUDViewChangeAnimationDuration animations:^{
+				[hudView setTitle:[change objectForKey:NSKeyValueChangeNewKey]];
```

Comment:
Hm, setting `hud.title` to `nil` gives us `NSNull` in this dictionary. :crying_cat_face: